### DISCUSSION TITLE
Versioning of releases

### DISCUSSION THREAD

**Discussion by goldstein0101** [databrickslabs/dbx#596] [Ideas]
Hey, do u recommend any resources for versioning when triggering release pipeline?

**Comment by renardeinside** [comment]
hey @goldstein0101 , 
there are various ways to do that. the default template suggests using git tags functionality to mark releases. 
If you're looking for python package versioning, it will be different depending on your versioning approach. 
Take a look at:
- https://pypi.org/project/setuptools-scm/
- https://pypi.org/project/poetry-dynamic-versioning/
- https://pypi.org/project/versioneer/0.10/
- https://github.com/mtkennerly/dunamai

And many-many others. There is no golden standard here, unfortunately.

**Reply by goldstein0101** [reply]
Yeah, since project is not maintaining some API, its a ML project where i will have new release if i for example change how im reading data,
so wondering how to version this stuff..
